### DISCUSSION TITLE
Recommended Ubuntu Server Version for Jamulus

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#2425] [Technical Support & Help] [answered]
Hello,

which Ubuntu Server Version is recommended for Jamulus installation?

My Jamulus currently runs on Ubuntu 18.04.6

Ubunty 22.04.LTS supported?

Thanks

**Comment by pljones** [comment, marked as answer]
The server has no particular dependencies.  Run on the best-supported OS version you have.

**Reply by reinhardwh** [reply]
Hello @pljones 
Thanks for confirmation!

Maybe this info should be added to server install tips.

When I started to work with Jamulus I read somewhere only 18.x.x is supported

**Reply by pljones** [reply]
We've got "improve the server documentation" down on the list of things we're hoping to address.

**Reply by gilgongo** [reply]
> When I started to work with Jamulus I read somewhere only 18.x.x is supported

One of our problems as been that until recently, there hasn't been much "official" documentation for Jamulus, so information about it was spread out over the Internet in various forms. This probably accounts for you reading that somewhere :-)

**Reply by reinhardwh** [reply]
Just updated my server from

Ubuntu 18.04.6

to

root@rv2575:~# lsb_release -a
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.4 LTS
Release:        20.04
Codename:       focal

Works perfectly

**Reply by reinhardwh** [reply]
This info should be made "official" on Jamulus Web page

**Reply by hoffie** [reply]
Do you have suggestions where exactly/how?

**Reply by reinhardwh** [reply]
Maybe on page:

https://jamulus.io/wiki/Server-Linux

**Reply by pljones** [reply]
We'd need a "tested distributions" page...  That's more overhead.  At the moment, the page indicates "Ubuntu/debian" and that's accurate.

**Reply by reinhardwh** [reply]
Ok for me.
